lib/android/src/main/java/com/airbnb/android/react/maps/AirMapManager.java

```diff
@@ -119,7 +120,7 @@ public void setShowBuildings(AirMapView view, boolean showBuildings) {
         view.map.setBuildingsEnabled(showBuildings);
     }
 
-    @ReactProp(name = "showsIndoors", defaultBoolean = false)
+    @ReactProp(name = "showsIndoors", defaultBoolean = true)
```

Comment:
Why do you have to change the default value here? Sounds like this is unnecessary?